web/pdf_viewer.css

```
@@ -30,7 +31,8 @@
   :root {
     --pdfViewer-padding-bottom: 9px;
     --page-margin: 9px auto 0;
-    --page-border: none;
+    --page-border: 1px solid CanvasText;
+    --page-border-image: none;
     --spreadHorizontalWrapped-margin-LR: 4.5px;
```

*I've not tested, but simply reading the surrounding the code:*
Don't you need to change this values, to account for the fact that there's now a border added just above?
Comparing with the "normal" rules, should this be updated to e.g. the following now?
```suggestion
    --spreadHorizontalWrapped-margin-LR: 3.5px;
```